plotly/offline/__init__.py

```diff
@@ -7,5 +7,8 @@
     download_plotlyjs,
     init_notebook_mode,
     iplot,
-    plot
+    iplot_mpl,
+    plot,
+    plot_mpl,
+    plotly_takeover
```

Comment:
Any of these perhaps?

* `take_mpl_offline`
* `make_mpl_offline`
* `convert_mpl_to_offline`

Neither `plotly_takeover` nor `plotly_mpl_takeover` makes all that much sense to me on the first pass.